guide/how-to/contribute.qmd

```diff
@@ -3,12 +3,40 @@ title: "Contribute"
 subtitle: "How to contribute to KIMMDY"
 ---
 
+## Setup
+
+See [Installation](./install-kimmdy.qmd). 
+
+For new plugins, make a separate repo, and add your entrypoint in the 
+`pyproject.toml` to the `kimmdy.reaction_plugins` entrypoint:
+
+```pyproject.toml
+[project.entry-points."kimmdy.reaction_plugins"]
+my_reaction = "kimmdy_my_reaction.reaction:My_Reaction_Class"
+```
+
+
+## Code Style
+
+Doc strings in numpy style, formatting using `black`.
+
+## Docs and Examples
+
+You can add labels to PR's to trigger CI to rebuild the documentation 
+or the `examples` directory. Just add the `docthis` or `build_examples` label.
+
+
+## PR's
+
+If a PR adds multiple commits, we use the `squash and merge` option to keep 
+the commit history clean. If a PR unavoidably contains changes the message 
+can be edited before merging to ensure `release please` picks up all changes.
+
```

Comment:
small note on that: Release Please can be super finicky with multiple entries in one commit message and often doesn't pick them up. Because of that I have mostly moved to rebasing for such cases, leading to individual commits for each feature. Optionally also with cleaning up a branch/PR via an interactive rebase or selectively committing changes from a reset locally .